storybook/stories/core/components/ListTable/ListSkeleton.story.vue

```diff
@@ -0,0 +1,167 @@
+<template>
```

Comment:
Code cannot be copied in the List Table copy option in the story book.

Reply:
Done